### ISSUE TITLE
cache posts and friends

### ISSUE THREAD

**Issue by jancborchardt** [agical/friendsunhosted#2]
To not present users with a complete blank slate (or only their own data) on every use, the fetched posts and list of friends should be cached in localStorage.

Until the new updates roll in (with a spinner displaying #1), you can still check out old posts.

**Comment by brolund** [comment]
Good idea. ATM we only store the remoteStorage access data there. Maybe we should even have that as the local master, and letting the GUI listen to what is in localStorage? My impression is that localStorage is pretty well supported among browsers. Is that right?

**Comment by jancborchardt** [comment]
Yeah, we don’t want to focus on localStorage too much, especially because stuff like IndexedDB (and that other thing) will make the concept of local browser storage more powerful.

It’s your call how you want to do it with the app though. :) Caching all the data in localStorage would work since it’s just all text at the moment – and then also poll the remoteStorages continuously for updates.
